torch/ao/nn/quantized/modules/functional_modules.py

```diff
@@ -220,6 +231,13 @@ def add_relu(self, x: Tensor, y: Tensor) -> Tensor:
         r = self.activation_post_process(r)
         return r
 
+    r"""Operation equivalent to ``torch.ops.quantized.matmul(Tensor, Tensor)``"""
+    def matmul(self, x: Tensor, y: Tensor) -> Tensor:
+        r = ops.quantized.matmul(x, y, scale=self.scale, zero_point=self.zero_point)
+        # Note: this operation is not observed because the observation is not
+        # needed for the quantized op.
```

Comment:
you can also remove this comment I think, it is observed, but we don't need to call `activation_post_process` in quantized model